org.eclipse.lemminx/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -285,6 +285,17 @@
 			"parameterTypes": []
 		}]
 	},
+	{
+		"name": "org.eclipse.lemminx.extensions.contentmodel.settings.XMLNamespacesSettings",
+		"methods": [{
+			"name": "<init>",
+			"parameterTypes": []
+		}]
```

Comment:
The namespace settings don't work in the native binary. If you add `"allDeclaredFields": true` in this JSON object, it will work.